examples/ruby/ruby_client.rb

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env ruby
+
+# Load the gem
+require 'openapi_client'
+
+CERT_DIR = 'config/https'
+SSL_CERT_FILE = 'ca.crt'
+CONJUR_CERT_FILE = 'conjur.crt'
+CONJUR_KEY_FILE = 'conjur.key'
+
+ACCOUNT="dev"
+LOGIN="admin"
+ADMIN_API_KEY=ENV["CONJUR_ADMIN_API_KEY"]
+
+# Setup client configuration
+config = OpenapiClient.configure
+config.scheme = "https"
+config.host = "localhost"
+# config.debugging = true
+config.username = LOGIN
+config.password = ADMIN_API_KEY
+config.verify_ssl = true
+config.ssl_ca_cert = File.join(CERT_DIR, SSL_CERT_FILE)
+config.cert_file = File.join(CERT_DIR, CONJUR_CERT_FILE)
+config.key_file = File.join(CERT_DIR, CONJUR_KEY_FILE)
+
+# Authenticating admin using basicAuth
+authn_instance = OpenapiClient::AuthnApi.new
+token = nil
+puts "Authenticating admin..."
+begin
+  token = authn_instance.authenticate(account=ACCOUNT, login=LOGIN, body=ADMIN_API_KEY, opts={accept_encoding: 'base64'})
+  puts "Base64 encoded token: #{token}"
+rescue OpenapiClient::ApiError => e
+  puts "Exception when calling AuthnApi->authenticate: #{e}"
+  exit(1)
+end
+
+# Change admin password using basicAuth
+new_password = "N3w-Passw0rd!"
+puts "\nChanging admin password..."
+begin
+  authn_instance.set_password(account=ACCOUNT, body=new_password)
+  puts "Password change successful."
+rescue OpenapiClient::ApiError => e
+  puts "Exception when calling AuthnApi->set_password: #{e}"
+  exit(1)
+end
+
+config.password = new_password
+
+# Add Conjur Token header to api configuration
+token_body = 'token="%s"' % [token]
+config.api_key['Authorization'] = token_body
+config.api_key_prefix['Authorization'] = 'Token'
+
+policy_instance = OpenapiClient::PoliciesApi.new
+authn_instance = OpenapiClient::AuthnApi.new
+
+# Load empty policy, allows the example to be run multiple times sequentially
+# Loading a policy returns data for users CREATED when the policy is loaded. Without loading 
+# an "empty" policy, if the user alice already exists due to a prior example run, loading 
+# the full policy will not respond with alice's api key.
+puts "\nLoading empty root policy..."
+empty_policy = "- !user admin"
```

Comment:
Since this is a constant too of sort, it can probably go up top. Now that I'm thinking about this, I think we may want to consider extracting the policy files into real files since they are the same across python and ruby examples (and we may have even more)